RHIDP-3393 - Document how users can access Plugin Marketplace in RHDH (#995)
redhat-developer/red-hat-developers-documentation-rhdh: modules/dynamic-plugins/proc-catalog-viewing.adoc

[id="rhdh-extensions-plugins-viewing_{context}"]
= Viewing available plugins

> Using Extensions, you can view plugins that are available for your {product} application.

Reviewer: ```suggestion
You can view plugins available for your {product} application on the *Extensions* page.
```
